src/kili/services/asset_import/base.py

```diff
@@ -177,25 +192,48 @@ def import_batch(self, assets: List[AssetLike]):
             assets = self.upload_local_content_to_bucket(assets)
         return super().import_batch(assets)
 
+    def get_content_type_and_data_from_content(
+        self, content: Union[str, None]
+    ) -> Tuple[bytes, Union[str, None]]:
+        """
+        Returns the data of the content (path) and its content type
+        """
+        assert content
+        with open(content, "rb") as file:
```

Comment:
Can you use pathlib? we are migrating away from `os.path`
here:
```suggestion
        with Path(content).open("rb") as file:
```

Reply:
done